README.md

```diff
@@ -54,11 +60,11 @@ skunkworks-crow app can be downloaded and installed from the link provided below
 [Download APK](https://github.com/opendatakit/skunkworks-crow/releases)
 
 ## 3. Using skunkworks-crow<a name="using-skunkworks-crow"></a>
-Share app can be used by enumerators in sharing forms and by supervisor in reviewing the forms.
+Skunkworks-crow app can be used by enumerators in sharing forms and by supervisor in reviewing the forms.
 
 ### 3.1 Viewing category of forms<a name="view-category"></a>
 
-Launch the app to see  a listing of all distinct versions of blank forms on the home screen. Tapping on any form will open a screen with four different tabs showing the statistics, sent, received and reviewed.
+Launch the app to see a listing of all distinct versions of blank forms on the home screen. Tapping on any form will open a screen with four different tabs showing the statistics, sent, received and reviewed.
```

Comment:
How about rephrasing it like, "On launching the app, the first screen that we see contains a list of forms available in ODK Collect. Tapping on any form opens a screen with 4 tabs - statistics, sent, received, and reviewed"

Reply:
Sounds good to me.